jsHelper/spicetifyWrapper.js

```diff
@@ -393,7 +393,7 @@ window.Spicetify = {
 			PanelContent:
 				modules.find(m => m?.render?.toString().includes("scrollBarContainer")) ||
 				functionModules.find(m => m.toString().includes("scrollBarContainer")),
-			PanelSkeleton: functionModules.find(m => m.toString().includes("label") && m.toString().includes("aside")),
+			PanelSkeleton: functionModules.find(m => m.toString().includes("label") && m.toString().includes("aside")) || modules.find(m => m?.render?.toString().includes('"section"')),
```

Comment:
@kyrie25 are you sure the double quoting is correct?
```suggestion
			PanelSkeleton: functionModules.find(m => m.toString().includes("label") && m.toString().includes("aside")) || modules.find(m => m?.render?.toString().includes("section")),
```

Reply:
Yes, `"section"` as in the literal tag name string `"section"`